editor/plugins/tiles/tile_set_editor.cpp

```diff
@@ -126,6 +100,43 @@ bool TileSetEditor::_can_drop_data_fw(const Point2 &p_point, const Variant &p_da
 	return false;
 }
 
+void TileSetEditor::_load_texture_files(const Vector<String> &p_paths) {
+	int source_id = TileSet::INVALID_SOURCE;
+	Vector<Ref<TileSetAtlasSource>> atlases;
+
+	for (String p_path : p_paths) {
```

Comment:
```suggestion
	for (const String &p_path : p_paths) {
```
Avoid referencing if not needed

Reply:
I really need to pay attention to this, sorry

Reply:
It's no problem, it's a learning experience